File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    vector<string> split(string s)
    {
        vector<string> result;
        istringstream iss(s);
        string word;
        while( iss >> word)
        {
            result.push_back(word);
        }
        return result;
    }
    int lengthOfLastWord(string s) {
        
        vector<string> ans=split(s);
        int size=ans.size();
        string last=ans[size-1];
        return last.length();
        
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        st = s.strip()
        for i in range(len(st)-1,0,-1):
            if st[i] == " ":
                return count
            else:
                count+=1
        return count 
